src/packages/data-type/components/ref-data-type/ref-data-type.element.ts

```diff
@@ -30,6 +33,9 @@ export class UmbRefDataTypeElement extends UmbElementMixin(UUIRefNodeElement) {
 						this.name = dataType.name ?? '';
 						this.propertyEditorUiAlias = dataType.editorUiAlias ?? '';
 						this.propertyEditorSchemaAlias = dataType.editorAlias ?? '';
+						if (this.propertyEditorUiAlias) {
```

Comment:
Want is was looking for here, was not only to just for this but also so do this: ```dataType.editorUiAlias ?? '' !== this.propertyEditorUiAlias```

and then move ```this.propertyEditorUiAlias = dataType.editorUiAlias ?? '';```inside the If-sentence.

Reply:
So getIconFromUiAlias is only called if that specific property did change. Otherwise we would load the icon again if another property of the manifest got changed. And we do only want to trigger an actual change of what effects the icon.. aka. the propertyEditorUiAlias.

Reply:
Ah I think i get it now , thank you 😄